README.md

```diff
@@ -170,6 +170,4 @@ I would like to thank microverse for this opportunity.
 

```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your commit messages capitalized to make your pull request descriptive and professional :+1: